Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -3028,9 +3030,9 @@ auto RenderLayerCompositor::attachWidgetContentLayersIfNecessary(RenderWidget& r
             return result;
 
         auto* contentsRenderView = frameContentsRenderView(renderer);
-        if (auto frameRootScrollingNodeID = contentsRenderView->frameView().scrollingNodeID()) {
+        if (auto frameRootScrollingNodeID = contentsRenderView->protectedFrameView()->scrollingNodeID()) {
```

Comment:
Again, can we store contentsRenderView->frameView() in a local variable instead of calling protectedFrameView() twice in succession?

Reply:
If you mean the call below, notice that it's actually `m_renderView`'s frame view, not `contentsRenderView`.